yt/yt/server/node/exec_node/job.cpp

```diff
@@ -2563,13 +2563,21 @@ TJobProxyInternalConfigPtr TJob::CreateConfig()
         }
     }
 
+    auto calculateShardingKey = [&] (size_t shardingKeyLength) {
+        auto randomPart = GetId().Underlying().Parts64[1];
+        auto shardingKey = Sprintf("%016lx", randomPart);
```

Comment:
Could we use `Format(..., randomPart)` here? 

`Format` – is is commonly used function for formatting in YTsaurus codebase.